### DISCUSSION TITLE
How to get "examples" running in my computer?

### DISCUSSION THREAD

**Discussion by linghuccc** [rainbow-me/rainbowkit#1823] [Q&A]
Step 1:
I used "git clone https://github.com/rainbow-me/rainbowkit.git" to download the whole project. 

Step 2:
And then copied "with-next-app" project in "examples" folder.

Step 3:
I tried to run "npm install" in "with-next-app" directory so that I could start the server by running "npm run dev" command. But "npm install" gave me following error message.

[2024-03-04T13_29_01_510Z-debug-0.log](https://github.com/rainbow-me/rainbowkit/files/14482144/2024-03-04T13_29_01_510Z-debug-0.log)

**Comment by 0xmkh** [comment]
@linghuccc You need to use pnpm instead of npm since our workspace is only dependent on pnpm. 

Can you try this ? 
- `pnpm install`
- `cd examples && cd with-next-app && pnpm dev`

**Reply by 0xmkh** [reply]
Also use pnpm version `8.14.1`

**Reply by linghuccc** [reply]
Thank you for your prompt reply. Yes, it's working now!
You are the best, man!
